packages/flutter_tools/lib/src/xcode_project.dart

```diff
@@ -21,18 +21,47 @@ import 'template.dart';
 ///
 /// This defines interfaces common to iOS and macOS projects.
 abstract class XcodeBasedProject extends FlutterProjectPlatform  {
-  static const String _hostAppProjectName = 'Runner';
+  final String _defaultHostAppName = 'Runner';
```

Comment:
Prefer const over final if the value is known at compilation time.
```suggestion
  static const String _defaultHostAppName = 'Runner';
```

Reply:
If it's `static` that means it's on the class, not instance, you need to call as `XcodeBasedProject._defaultHostAppName`.  I didn't include that suggestion but this by itself won't compile.